packages/venia-concept/src/components/Head/headProvider.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { HeadProvider as _HeadProvider } from 'react-head';
+
+const filterOutTitleTags = titleTags => {
+    titleTags
+        .filter(titleNode => !titleNode.dataset.synteticTag)
+        .forEach(titleTag => {
+            if (titleTag) titleTag.parentNode.removeChild(titleTag);
```

Comment:
Did this pass `prettier` formatting?

`yarn run prettier`

Reply:
Yeah, it did. Does it mean I am running a different prettier config than yours?

Reply:
No, you're right - it worked for me too. Just haven't seen very many of these I guess 🤷‍♂